packages/tools/kadena-cli/src/tx/utils/txHelpers.ts

```diff
@@ -81,7 +83,15 @@ export async function getTransactions(
           if (content === null) return null;
           // signed=false can still return already signed transactions
           const schema = signed ? ICommandSchema : IUnsignedCommandSchema;
-          const parsed = schema.safeParse(JSON.parse(content));
+          const JSONParsedContent = JSON.parse(content) as
+            | ICommand
+            | IUnsignedCommand;
+          const parsed = schema.safeParse(JSONParsedContent);
+          if (parsed.success && !signed && !all) {
+            const isSigned = JSONParsedContent.sigs.every((sig) => !!sig);
```

Comment:
would prefer to just introduce a 3rd schema variant to have `ICommandSchema` and `IUnsignedCommandSchema`. The problem now was that `IUnsignedCommandSchema` also accepts signed commands.

If it's in a rush I can accept this for now.